src/common/components/DataTable/DataTableSearchAndFilters.tsx

```diff
@@ -87,7 +87,7 @@ export const DataTableSearchAndFilters: FC<DataTableSearchAndFilterProps> = ({
   };
 
   const removeFilter = (attr: string, op: FilterOp) => {
-    setActiveFilters(activeFilters.filter(filter => filter.attr !== attr && filter.op !== op));
+    setActiveFilters(activeFilters.filter(filter => filter.attr !== attr));
```

Comment:
Are we sure `filter.op !== op` was not there for good reason? Is this really necessary to remove?

Reply:
Nice catch. I found that the RecentDateFilter and EnumFilter makes use of the op parameter (besides everything else in usePSF and etc). I'll re-add it and see if I can find what is causing the aforementioned issue.

Reply:
On second thought, those filters make use of the `op` parameter of the removeFilter method. That method isn't being edited here. The only thing this affects are the active filters. I'm not seeing any negative consequences with this change in place based on the testing that I have done so far.

Reply:
@joshwingreene The logic of the old filter was incorrect, I pushed a fix correcting it to be `!(filter.attr === attr && filter.op === op)`